Dockerfile.windows

```diff
@@ -166,7 +166,7 @@ FROM microsoft/windowsservercore
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 ARG GO_VERSION=1.17.3
-ARG CONTAINERD_VERSION=1.5.7
+ARG CONTAINERD_VERSION=1.6.0-beta.2
```

Comment:
Note to self; add temporary fix for version string / download URL in Windows Dockerfile